canvas_modules/common-canvas/src/common-properties/actions/image/image.jsx

```diff
@@ -40,58 +43,52 @@ class ImageAction extends React.Component {
 		}
 	}
 
+	renderIcon() {
+		const icon = (<img
+			src={this.props.action.image.url}
+			aria-label={this.props.action.name}
+			height={this.height}
+			width={this.width}
+		/>);
+		return icon;
+	}
+
 	render() {
-		let height = {};
-		let width = {};
-		if (this.props.action.image.size) {
-			height = this.props.action.image.size.height ? { "height": this.props.action.image.size.height } : {};
-			width = this.props.action.image.size.width ? { "width": this.props.action.image.size.width } : {};
-		}
 
 		const disabled = this.props.state === STATES.DISABLED;
 		const customClassName = this.props.action.className ? this.props.action.className : "";
 		const className = classNames("properties-action-image", { "left": this.props.action.image.placement === "left" },
 			{ "right": this.props.action.image.placement === "right" }, { "hide": this.props.state === STATES.HIDDEN },
 			{ "disabled": disabled }, customClassName);
+		if (this.props.action.image.size) {
+			this.height = this.props.action.image.size.height + "px";
+			this.width = this.props.action.image.size.width + "px";
+		}
 
 		const image = (
-			<div data-id={this.props.action.name}>
-				<img
-					src={this.props.action.image.url}
+			<div
+				style={{
+					width: this.width,
+					height: this.height
+				}}
+			>
+				<Button
+					// Use description to hide carbon tooltip
+					hasIconOnly={Boolean(this.props.action.description?.text)}
```

Comment:
Why does description determine if it's iconOnly?  A better comment here would be good.

Reply:
As per Carbon [Button Documentation](https://react.carbondesignsystem.com/?path=/docs/components-button--overview#button-hasicononly) hasIconOnly allows tooltip to be shown for the images which doesn't have description.text empty tooltip would appear like below.

https://github.com/user-attachments/assets/ec2cfa6a-fb4b-4471-b39b-584b9485e0c2